geomesa-tools/bin/geomesa-accumulo

```diff
@@ -0,0 +1,251 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+setGeoHome ()
+{
+    SOURCE="${BASH_SOURCE[0]}"
+    # resolve $SOURCE until the file is no longer a symlink
```

Comment:
This looks like code I've seen in a number of other places...  If we copied this from somewhere, we should probably give attribution.